### PR TITLE
Converts all necessary << outputs into the to_chat() macro.

### PR DESCRIPTION
**Pull request by LordFowl**
This PR will lead us towards the Promised Day, for in its wake there shall be much celebration and ecstasy as this world becomes a world suitable for developer hegemony. The first strike is thusly;

All << is converted into to_chat().

Reduces over 8,000 instances of << to the 700 instances of it that are not output operators.
Some instances may still lurk, primarily messages like this:
`random_variable_fowl_could_neverthink_of.owner << message_variable_not_a_direct_string`
On the same basis, the biggest danger is some things that should NOT be to_chat now being to_chat. This is primarily things like sound(), and while I have taken great care to avoid clobbering them some that follow a similar format as above might have been trampled.

- [x] Convert all "user <<"
- [x] Convert all "M/H/F/U/C/K/etc"
- [x] Convert everything in the middle
- [x] Update to stable BYOND 512 so the compiler stops being noisy

### CODE REVIEW

**Comment by Chada-1** [comment]
If this does what I think it does, I'm a big fan. What exactly it does I still have no clue.

**Comment by Sindorman** [comment]
I feel like this is a good test merge example

**Comment by LordFowl** [comment]
!review

File: code/game/area/areas.dm [hunk 1]
```diff
@@ -334,23 +334,23 @@ var/list/mob/living/forced_ambiance_list = new
...
 	if(!L.client.ambience_playing)
 		L.client.ambience_playing = 1
-		L << sound('sound/ambience/shipambience.ogg', repeat = loop_ambience, wait = 0, volume = 35, channel = 2)
+		to_chat(L, sound('sound/ambience/shipambience.ogg', repeat = loop_ambience, wait = 0, volume = 35, channel = 2))
 
 	if(forced_ambience)
 		if(forced_ambience.len)
 			forced_ambiance_list |= L
-			L << sound(pick(forced_ambience), repeat = loop_ambience, wait = 0, volume = 25, channel = 1)
+			to_chat(L, sound(pick(forced_ambience), repeat = loop_ambience, wait = 0, volume = 25, channel = 1))
```
Comment (Arrow768) [inline code/game/area/areas.dm]: sound

File: code/game/area/areas.dm [hunk 2]
```diff
@@ -334,23 +334,23 @@ var/list/mob/living/forced_ambiance_list = new
...
 		else
-			L << sound(null, channel = 1)
+			to_chat(L, sound(null, channel = 1))
```
Comment (Arrow768) [inline code/game/area/areas.dm]: sound

File: code/game/area/areas.dm [hunk 3]
```diff
@@ -334,23 +334,23 @@ var/list/mob/living/forced_ambiance_list = new
...
 	else if(src.ambience.len && prob(35))
 		if((world.time >= L.client.played + 600))
 			var/sound = pick(ambience)
-			L << sound(sound, repeat = 0, wait = 0, volume = 25, channel = 1)
+			to_chat(L, sound(sound, repeat = 0, wait = 0, volume = 25, channel = 1))
```
Comment (Arrow768) [inline code/game/area/areas.dm]: sound

File: code/game/machinery/kitchen/cooking_machines/fryer.dm [hunk 4]
```diff
@@ -190,10 +190,12 @@
 
 
 		if(!nopain)
-			victim << "<span class='danger'>Agony consumes you as searing hot oil scorches your [E ? E.name : "flesh"] horribly!</span>"
+			var/your_lie_in_april = E ? E.name : "flesh"
```
Comment (Arrow768) [inline code/game/machinery/kitchen/cooking_machines/fryer.dm]: var_name

File: code/game/machinery/kitchen/cooking_machines/fryer.dm [hunk 5]
```diff
@@ -190,10 +190,12 @@
...
+			to_chat(victim, "<span class='danger'>Agony consumes you as searing hot oil scorches your [your_lie_in_april] horribly!</span>")
 			victim.emote("scream")
 		else
-			victim << "<span class='danger'>Searing hot oil scorches your [E ? E.name : "flesh"]!</span>"
+			var/welcome_to_the_nhk = E ? E.name : "flesh"
```
Comment (Arrow768) [inline code/game/machinery/kitchen/cooking_machines/fryer.dm]: var_name

File: code/game/machinery/teleporter.dm [hunk 6]
```diff
@@ -254,7 +254,7 @@
 	if (istype(M, /mob/living))
 		var/mob/living/MM = M
 		if(MM.check_contents_for(/obj/item/weapon/storage/backpack/holding))
-			MM << "<span class='warning'>The Bluespace interface on your Bag of Holding interferes with the teleport!</span>"
+			Mto_chat(M, "<span class='warning'>The Bluespace interface on your Bag of Holding interferes with the teleport!</span>")
```
Comment (Arrow768) [inline code/game/machinery/teleporter.dm]: issue

File: code/game/machinery/teleporter.dm [hunk 7]
```diff
@@ -227,7 +227,7 @@
 	if (istype(M, /mob/living))
 		var/mob/living/MM = M
 		if(MM.check_contents_for(/obj/item/weapon/disk/nuclear))
-			MM << "<span class='warning'>Something you are carrying seems to be unable to pass through the portal. Better drop it if you want to go through.</span>"
+			Mto_chat(M, "<span class='warning'>Something you are carrying seems to be unable to pass through the portal. Better drop it if you want to go through.</span>")
```
Comment (Arrow768) [inline code/game/machinery/teleporter.dm]: issue

File: code/game/mecha/combat/combat.dm [hunk 8]
```diff
@@ -232,7 +232,7 @@
 							</body>
 							</html>
 						  "}
-		occupant << browse(window, "window=sam;size=800x600;")
+		to_chat(occupant, browse(window, "window=sam;size=800x600;"))
```
Comment (Arrow768) [inline code/game/mecha/combat/combat.dm]: browse

File: code/game/mecha/combat/marauder.dm [hunk 9]
```diff
@@ -170,7 +170,7 @@
 		src.occupant_message("<font color='[src.zoom?"blue":"red"]'>Zoom mode [zoom?"en":"dis"]abled.</font>")
 		if(zoom)
 			src.occupant.client.view = 12
-			src.occupant << sound('sound/mecha/imag_enh.ogg',volume=50)
+			src.to_chat(occupant, sound('sound/mecha/imag_enh.ogg',volume=50))
```
Comment (Arrow768) [inline code/game/mecha/combat/marauder.dm]: sound

File: code/game/mecha/equipment/mecha_equipment.dm [hunk 10]
```diff
@@ -46,9 +46,9 @@
 		chassis.occupant_message("<font color='red'>The [src] is destroyed!</font>")
 		chassis.log_append_to_last("[src] is destroyed.",1)
 		if(istype(src, /obj/item/mecha_parts/mecha_equipment/weapon))
-			chassis.occupant << sound('sound/mecha/weapdestr.ogg',volume=50)
+			chassis.to_chat(occupant, sound('sound/mecha/weapdestr.ogg',volume=50))
```
Comment (Arrow768) [inline code/game/mecha/equipment/mecha_equipment.dm]: invalid, sound

File: code/game/mecha/equipment/mecha_equipment.dm [hunk 11]
```diff
@@ -46,9 +46,9 @@
 		chassis.occupant_message("<font color='red'>The [src] is destroyed!</font>")
 		chassis.log_append_to_last("[src] is destroyed.",1)
 		if(istype(src, /obj/item/mecha_parts/mecha_equipment/weapon))
-			chassis.occupant << sound('sound/mecha/weapdestr.ogg',volume=50)
+			chassis.to_chat(occupant, sound('sound/mecha/weapdestr.ogg',volume=50))
 		else
-			chassis.occupant << sound('sound/mecha/critdestr.ogg',volume=50)
+			chassis.to_chat(occupant, sound('sound/mecha/critdestr.ogg',volume=50))
```
Comment (Arrow768) [inline code/game/mecha/equipment/mecha_equipment.dm]: invalid, sound

File: code/game/mecha/equipment/tools/medical_tools.dm [hunk 12]
```diff
@@ -93,7 +93,7 @@
 	if(old_filter.get("eject"))
 		go_out()
 	if(old_filter.get("view_stats"))
-		chassis.occupant << browse(get_occupant_stats(),"window=msleeper")
+		chassis.to_chat(occupant, browse(get_occupant_stats(),"window=msleeper"))
```
Comment (Arrow768) [inline code/game/mecha/equipment/tools/medical_tools.dm]: invalid, browse

File: code/game/mecha/equipment/tools/medical_tools.dm [hunk 13]
```diff
@@ -347,7 +347,7 @@
 			log_message("Reagent processing started.")
 		return
 	if(old_filter.get("show_reagents"))
-		chassis.occupant << browse(get_reagents_page(),"window=msyringegun")
+		chassis.to_chat(occupant, browse(get_reagents_page(),"window=msyringegun"))
```
Comment (Arrow768) [inline code/game/mecha/equipment/tools/medical_tools.dm]: invalid, browse

File: code/game/mecha/mecha.dm [hunk 14]
```diff
@@ -514,7 +514,7 @@
 	internal_damage |= int_dam_flag
 	start_process(MECHA_PROC_DAMAGE)
 	log_append_to_last("Internal damage of type [int_dam_flag].",1)
-	occupant << sound('sound/machines/warning-buzzer.ogg',wait=0)
+	to_chat(occupant, sound('sound/machines/warning-buzzer.ogg',wait=0))
```
Comment (Arrow768) [inline code/game/mecha/mecha.dm]: sound

File: code/game/mecha/mecha.dm [hunk 15]
```diff
@@ -1245,7 +1245,7 @@
 	set popup_menu = 0
 	if(usr!=src.occupant)
 		return
-	src.occupant << browse(src.get_stats_html(), "window=exosuit")
+	src.to_chat(occupant, browse(src.get_stats_html(), "window=exosuit"))
```
Comment (Arrow768) [inline code/game/mecha/mecha.dm]: invalid, browse

File: code/game/mecha/mecha.dm [hunk 16]
```diff
@@ -1276,7 +1276,7 @@
 		src.log_message("[mob_container] moved out.")
 		occupant.reset_view()
 
-		src.occupant << browse(null, "window=exosuit")
+		src.to_chat(occupant, browse(null, "window=exosuit"))
```
Comment (Arrow768) [inline code/game/mecha/mecha.dm]: invalid, browse

File: code/game/mecha/mecha.dm [hunk 17]
```diff
@@ -1641,7 +1641,7 @@
 		return
 	if (href_list["view_log"])
 		if(usr != src.occupant)	return
-		src.occupant << browse(src.get_log_html(), "window=exosuit_log")
+		src.to_chat(occupant, browse(src.get_log_html(), "window=exosuit_log"))
```
Comment (Arrow768) [inline code/game/mecha/mecha.dm]: invalid, browse

File: code/game/mecha/mecha.dm [hunk 18]
```diff
@@ -2145,7 +2145,7 @@
  					   </body>
 						</html>"}
 
-	occupant << browse(output, "window=ex_debug")
+	to_chat(occupant, browse(output, "window=ex_debug"))
```
Comment (Arrow768) [inline code/game/mecha/mecha.dm]: browse

File: code/game/machinery/teleporter.dm (hunk 6 above)
Comment (LordFowl) [reply code/game/machinery/teleporter.dm]: Would probably be best just to delete the commentedout code.

File: code/_helpers/icons.dm [hunk 19]
```diff
@@ -349,7 +349,7 @@ proc/ReadRGB(rgb)
 					++which
 				else if(!(digits & 1)) ++which
 			if(1)
-				g = (g << 4) | ch
+				g = (g << 4 | ch)
```
Comment (skull132) [inline code/_helpers/icons.dm]:
```suggestion
				g = (g << 4) | ch
```

This is bad.

**Comment by LordFowl** [comment]
!review

**Comment by skull132** [comment]
I think I'm going to test-merge this tomorrow/today for a round, and see if anything horrible implodes.

File: code/datums/api.dm [hunk 20]
```diff
@@ -839,12 +839,12 @@ proc/api_update_command_database()
 	C.received_discord_pm = world.time
 	C.discord_admin = queryparams["senderkey"]
 
-	C << 'sound/effects/adminhelp.ogg'
-	C << message
+	to_chat(C, 'sound/effects/adminhelp.ogg')
```
Comment (Alberyk) [inline code/datums/api.dm]:
```suggestion
	sound_to(C, 'sound/effects/adminhelp.ogg')
```

File: code/game/sound.dm [hunk 21]
```diff
@@ -284,13 +284,13 @@ var/list/footstepfx = list("defaultstep","concretestep","grassstep","dirtstep","
 			var/area/A = get_area(src)
 			S.environment = A.sound_env
 
-	src << S
+	to_chat(src, S)
```
Comment (Alberyk) [inline code/game/sound.dm]:
```suggestion
	sound_to(src, S)
```

File: code/game/sound.dm [hunk 22]
```diff
@@ -284,13 +284,13 @@ var/list/footstepfx = list("defaultstep","concretestep","grassstep","dirtstep","
...
 
-	src << S
+	to_chat(src, S)
 	return S.volume
 
 /client/proc/playtitlemusic()
 	if(!SSticker.login_music)	return
 	if(prefs.toggles & SOUND_LOBBY)
-		src << sound(SSticker.login_music, repeat = 0, wait = 0, volume = 85, channel = 1) // MAD JAMS
+		src << sound(SSticker.login_music, repeat = 0, wait = 0, volume = 85, channel = 1) // MAD JAMS)
```
Comment (Alberyk) [inline code/game/sound.dm]: There is an extra ( here.

File: code/modules/admin/admin_verbs.dm [hunk 23]
```diff
@@ -1061,8 +1061,8 @@ var/list/admin_verbs_cciaa = list(
 	set desc = "Tells everyone to man up and deal with it."
 
 	for (var/mob/T as mob in mob_list)
-		T << "<br><center><span class='notice'><b><font size=4>Man up.<br> Deal with it.</font></b><br>Move on.</span></center><br>"
-		T << 'sound/voice/ManUp1.ogg'
+		to_chat(T, "<br><center><span class='notice'><b><font size=4>Man up.<br> Deal with it.</font></b><br>Move on.</span></center><br>")
+		to_chat(T, 'sound/voice/ManUp1.ogg')
```
Comment (Alberyk) [inline code/modules/admin/admin_verbs.dm]:
```suggestion
		sound_to(T, 'sound/voice/ManUp1.ogg')
```

File: code/modules/admin/verbs/adminhelp.dm [hunk 24]
```diff
@@ -127,12 +127,12 @@ var/list/adminhelp_ignored_words = list("unknown","the","a","an","of","monkey","
 			if(X.is_afk())
 				admin_number_afk++
 			if(X.prefs.toggles & SOUND_ADMINHELP)
-				X << 'sound/effects/adminhelp.ogg'
+				to_chat(X, 'sound/effects/adminhelp.ogg')
```
Comment (Alberyk) [inline code/modules/admin/verbs/adminhelp.dm]:
```suggestion
				sound_to(X, 'sound/effects/adminhelp.ogg')
```

File: code/modules/admin/verbs/adminpm.dm [hunk 25]
```diff
@@ -142,12 +142,12 @@
...
 		receiver_message += ": <span class='message'>[msg]</span>"
 	receiver_message += "</span></span>"
-	C << receiver_message
+	to_chat(C, receiver_message)
 
 	//play the recieving admin the adminhelp sound (if they have them enabled)
 	//non-admins shouldn't be able to disable this
 	if(C.prefs && C.prefs.toggles & SOUND_ADMINHELP)
-		C << 'sound/effects/adminhelp.ogg'
+		to_chat(C, 'sound/effects/adminhelp.ogg')
```
Comment (Alberyk) [inline code/modules/admin/verbs/adminpm.dm]:
```suggestion
		sound_to(C, 'sound/effects/adminhelp.ogg')
```

File: code/modules/admin/verbs/playsound.dm [hunk 26]
```diff
@@ -17,7 +17,7 @@ var/list/sounds_cache = list()
 	message_admins("[key_name_admin(src)] played sound [S]", 1)
 	for(var/mob/M in player_list)
 		if(M.client.prefs.toggles & SOUND_MIDI)
-			M << uploaded_sound
+			to_chat(M, uploaded_sound)
```
Comment (Alberyk) [inline code/modules/admin/verbs/playsound.dm]:
```suggestion
			sound_to(M, uploaded_sound)
```

File: code/modules/clothing/glasses/glasses.dm [hunk 27]
```diff
@@ -47,18 +47,18 @@ BLIND     // can't see anything
...
+			to_chat(usr, "You deactivate the optical matrix on the [src].")
 			if(activated_color)
 				set_light(0)
 		else
 			active = 1
 			icon_state = initial(icon_state)
 			user.update_inv_glasses()
 			if(activation_sound)
-				usr << activation_sound
+				to_chat(usr, activation_sound)
```
Comment (Alberyk) [inline code/modules/clothing/glasses/glasses.dm]:
```suggestion
				sound_to(usr, activation_sound)
```

File: code/modules/flufftext/Hallucination.dm [hunk 28]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
 
 			if(41 to 65)
 				//Strange audio
-				//src << "Strange Audio"
+				//to_chat(src, "Strange Audio")
 				switch(rand(1,12))
-					if(1) src << 'sound/machines/airlock.ogg'
+					if(1) to_chat(src, 'sound/machines/airlock.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
					if(1) sound_to(src, 'sound/machines/airlock.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 29]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
 
 			if(41 to 65)
 				//Strange audio
-				//src << "Strange Audio"
+				//to_chat(src, "Strange Audio")
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
				//sound_to(src, "Strange Audio")
```

File: code/modules/flufftext/Hallucination.dm [hunk 30]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
 					if(2)
-						if(prob(50))src << 'sound/effects/Explosion1.ogg'
-						else src << 'sound/effects/Explosion2.ogg'
-					if(3) src << 'sound/effects/explosionfar.ogg'
-					if(4) src << 'sound/effects/Glassbr1.ogg'
-					if(5) src << 'sound/effects/Glassbr2.ogg'
-					if(6) src << 'sound/effects/Glassbr3.ogg'
-					if(7) src << 'sound/machines/twobeep.ogg'
-					if(8) src << 'sound/machines/windowdoor.ogg'
+						if(prob(50))to_chat(src, 'sound/effects/Explosion1.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
						if(prob(50))sound_to(src, 'sound/effects/Explosion1.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 31]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
+						else to_chat(src, 'sound/effects/Explosion2.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
						else sound_to(src, 'sound/effects/Explosion2.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 32]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
+					if(3) to_chat(src, 'sound/effects/explosionfar.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
					if(3) sound_to(src, 'sound/effects/explosionfar.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 33]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
+					if(4) to_chat(src, 'sound/effects/Glassbr1.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
					if(4) sound_to(src, 'sound/effects/Glassbr1.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 34]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
+					if(5) to_chat(src, 'sound/effects/Glassbr2.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
					if(5) sound_to(src, 'sound/effects/Glassbr2.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 35]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
+					if(6) to_chat(src, 'sound/effects/Glassbr3.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
					if(6) sound_to(src, 'sound/effects/Glassbr3.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 36]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
+					if(7) to_chat(src, 'sound/machines/twobeep.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
					if(7) sound_to(src, 'sound/machines/twobeep.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 37]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
+					if(8) to_chat(src, 'sound/machines/windowdoor.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
					if(8) sound_to(src, 'sound/machines/windowdoor.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 38]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
 					if(9)
 						//To make it more realistic, I added two gunshots (enough to kill)
-						src << 'sound/weapons/Gunshot.ogg'
+						to_chat(src, 'sound/weapons/Gunshot.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
						sound_to(src, 'sound/weapons/Gunshot.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 39]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
 						spawn(rand(10,30))
-							src << 'sound/weapons/Gunshot.ogg'
-					if(10) src << 'sound/weapons/smash.ogg'
+							to_chat(src, 'sound/weapons/Gunshot.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
							sound_to(src, 'sound/weapons/Gunshot.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 40]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
+					if(10) to_chat(src, 'sound/weapons/smash.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
					if(10) sound_to(src, 'sound/weapons/smash.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 41]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
 					if(11)
 						//Same as above, but with tasers.
-						src << 'sound/weapons/Taser.ogg'
+						to_chat(src, 'sound/weapons/Taser.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
						sound_to(src, 'sound/weapons/Taser.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 42]
```diff
@@ -110,29 +110,29 @@ mob/living/carbon/proc/handle_hallucinations()
...
 						spawn(rand(10,30))
-							src << 'sound/weapons/Taser.ogg'
+							to_chat(src, 'sound/weapons/Taser.ogg')
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
							sound_to(src, 'sound/weapons/Taser.ogg')
```

File: code/modules/flufftext/Hallucination.dm [hunk 43]
```diff
@@ -141,10 +141,10 @@ mob/living/carbon/proc/handle_hallucinations()
 							'sound/hallucinations/growl3.ogg', 'sound/hallucinations/im_here1.ogg', 'sound/hallucinations/im_here2.ogg', 'sound/hallucinations/i_see_you1.ogg', 'sound/hallucinations/i_see_you2.ogg',\
 							'sound/hallucinations/look_up1.ogg', 'sound/hallucinations/look_up2.ogg', 'sound/hallucinations/over_here1.ogg', 'sound/hallucinations/over_here2.ogg', 'sound/hallucinations/over_here3.ogg',\
 							'sound/hallucinations/turn_around1.ogg', 'sound/hallucinations/turn_around2.ogg', 'sound/hallucinations/veryfar_noise.ogg', 'sound/hallucinations/wail.ogg')
-						src << pick(creepyasssounds)
+						to_chat(src, pick(creepyasssounds))
```
Comment (Alberyk) [inline code/modules/flufftext/Hallucination.dm]:
```suggestion
						sound_to(src, pick(creepyasssounds))
```

File: code/modules/mob/abstract/new_player/new_player.dm [hunk 44]
```diff
@@ -126,17 +126,17 @@ INITIALIZE_IMMEDIATE(/mob/abstract/new_player)
 			if(!client)	return 1
 			var/mob/abstract/observer/observer = new /mob/abstract/observer(src)
 			spawning = 1
-			src << sound(null, repeat = 0, wait = 0, volume = 85, channel = 1) // MAD JAMS cant last forever yo
+			src << sound(null, repeat = 0, wait = 0, volume = 85, channel = 1) // MAD JAMS cant last forever yo)
```
Comment (Alberyk) [inline code/modules/mob/abstract/new_player/new_player.dm]: extra )

File: code/modules/mob/abstract/new_player/new_player.dm [hunk 45]
```diff
@@ -421,7 +421,7 @@ INITIALIZE_IMMEDIATE(/mob/abstract/new_player)
 	else
 		client.prefs.copy_to(new_character)
 
-	src << sound(null, repeat = 0, wait = 0, volume = 85, channel = 1) // MAD JAMS cant last forever yo
+	src << sound(null, repeat = 0, wait = 0, volume = 85, channel = 1) // MAD JAMS cant last forever yo)
```
Comment (Alberyk) [inline code/modules/mob/abstract/new_player/new_player.dm]: Extra )

File: code/modules/mob/abstract/new_player/new_player.dm [hunk 46]
```diff
@@ -461,8 +461,8 @@ INITIALIZE_IMMEDIATE(/mob/abstract/new_player)
 	return 0
 
 /mob/abstract/new_player/proc/close_spawn_windows()
-	src << browse(null, "window=latechoices") //closes late choices window
-	src << browse(null, "window=playersetup") //closes the player setup window
+	src << browse(null, "window=latechoices") //closes late choices window)
+	src << browse(null, "window=playersetup") //closes the player setup window)
```
Comment (Alberyk) [inline code/modules/mob/abstract/new_player/new_player.dm]: Extra )

File: code/modules/mob/abstract/new_player/new_player.dm [hunk 47]
```diff
@@ -461,8 +461,8 @@ INITIALIZE_IMMEDIATE(/mob/abstract/new_player)
 	return 0
 
 /mob/abstract/new_player/proc/close_spawn_windows()
-	src << browse(null, "window=latechoices") //closes late choices window
-	src << browse(null, "window=playersetup") //closes the player setup window
+	src << browse(null, "window=latechoices") //closes late choices window)
```
Comment (Alberyk) [inline code/modules/mob/abstract/new_player/new_player.dm]: extra )

File: code/datums/diseases/advance/symptoms/fever.dm [hunk 48]
```diff
@@ -28,7 +28,7 @@ Bonus
 	..()
 	if(prob(SYMPTOM_ACTIVATION_PROB))
 		var/mob/living/carbon/M = A.affected_mob
-		M << "<span class='notice'>[pick("You feel hot.", "You feel like you're burning.")]</span>"
+		to_chat(M, "<span class='notice'>You feel like you're burning.</span>")
```
[the remark this answers is not in the document]
Comment (LordFowl) [reply code/datums/diseases/advance/symptoms/fever.dm]: _Oh well_

File: code/game/gamemodes/game_mode.dm [hunk 49]
```diff
@@ -693,13 +693,7 @@ proc/get_nt_opposed()
...
 /proc/show_generic_antag_text(var/datum/mind/player)
 	if(player.current)
-		player.current << \
-		"You are an antagonist! <font color=blue>Within the rules,</font> \
-		try to act as an opposing force to the crew. Further RP and try to make sure \
-		other players have <i>fun</i>! If you are confused or at a loss, always adminhelp, \
-		and before taking extreme actions, please try to also contact the administration! \
-		Think through your actions and make the roleplay immersive! <b>Please remember all \
-		rules aside from those without explicit exceptions apply to antagonists.</b>"
+		to_chat(player.current, "You are an antagonist! <font color=blue>Within the rules,</font> try to act as an opposing force to the crew. Further RP and try to make sure other players have <i>fun</i>! If you are confused or at a loss, always adminhelp, and before taking extreme actions, please try to also contact the administration! Think through your actions and make the roleplay immersive! <b>Please remember all rules aside from those without explicit exceptions apply to antagonists.</b>")
```
[the remark this answers is not in the document]
Comment (LordFowl) [reply code/game/gamemodes/game_mode.dm]: Multi-line function calls are more trouble than they're worth.

File: code/game/machinery/computer/security.dm [hunk 50]
```diff
@@ -131,8 +131,8 @@
 				if(3.0)
 					dat += "<CENTER><B>Security Record</B></CENTER><BR>"
 					if ((istype(active1, /datum/data/record) && data_core.general.Find(active1)))
-						user << browse_rsc(active1.fields["photo_front"], "front.png")
-						user << browse_rsc(active1.fields["photo_side"], "side.png")
+						to_chat(user, browse_rsc(active1.fields["photo_front"], "front.png"))
+						to_chat(user, browse_rsc(active1.fields["photo_side"], "side.png"))
```
[the remark this answers is not in the document]
Comment (LordFowl) [reply code/game/machinery/computer/security.dm]: No.

File: code/modules/flufftext/Hallucination.dm (hunk 29 above)
Comment (LordFowl) [reply code/modules/flufftext/Hallucination.dm]: This is text, bud.

File: code/modules/admin/verbs/getlogs.dm [hunk 51]
```diff
@@ -72,8 +72,8 @@
 		return
 
 	message_admins("[key_name_admin(src)] accessed file: [path]")
-	src << run( file(path) )
-	src << "Attempting to send file, this may take a fair few minutes if the file is very large."
+	to_chat(src, run( file(path) ))
```
Comment (skull132) [inline code/modules/admin/verbs/getlogs.dm]:
```suggestion
	src << run( file(path) )
```

Psure this is special.

File: code/modules/admin/verbs/getlogs.dm [hunk 52]
```diff
@@ -52,8 +52,8 @@
...
```
Comment (skull132) [inline code/modules/admin/verbs/getlogs.dm]:
```suggestion
	src << run( file(path) )
```

File: code/modules/admin/verbs/getlogs.dm [hunk 53]
```diff
@@ -87,9 +87,9 @@
 
 	var/path = "data/logs/[time2text(world.realtime,"YYYY/MM-Month/DD-Day")].log"
 	if( fexists(path) )
-		src << run( file(path) )
+		to_chat(src, run( file(path) ))
```
Comment (skull132) [inline code/modules/admin/verbs/getlogs.dm]:
```suggestion
		src << run( file(path) )
```

File: code/modules/admin/verbs/getlogs.dm [hunk 54]
```diff
@@ -102,10 +102,10 @@
 
 	var/path = "data/logs/[time2text(world.realtime,"YYYY/MM-Month/DD-Day")] Attack.log"
 	if( fexists(path) )
-		src << run( file(path) )
+		to_chat(src, run( file(path) ))
```
Comment (skull132) [inline code/modules/admin/verbs/getlogs.dm]:
```suggestion
		src << run( file(path) )
```

File: code/modules/admin/verbs/getlogs.dm [hunk 55]
```diff
@@ -102,10 +102,10 @@
...
 	else
-		src << "<font color='red'>Error: view_atk_log(): File not found/Invalid path([path]).</font>"
+		to_chat(src, "<font color='red'>Error: view_atk_log(): File not found/Invalid path([path]).</font>")
 		return
-	usr << run( file(path) )
+	to_chat(usr, run( file(path) ))
```
Comment (skull132) [inline code/modules/admin/verbs/getlogs.dm]:
```suggestion
	usr << run( file(path) )
```

**Pull request merged**